<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>969</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="matchPlan">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>581</width>
      <height>361</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1113</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <column>
     <property name="text">
      <string>Beginn</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Gast</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Heim</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ergebnis</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="finish_tournament">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>900</y>
      <width>115</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Finish Tournament</string>
    </property>
   </widget>
   <widget class="QLabel" name="contestor_label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>108</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Contestors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tournament_parameters">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>146</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Tournament Parameters</string>
    </property>
   </widget>
   <widget class="QPushButton" name="teams_and_schedule">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>46</y>
      <width>124</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Teams and Schedule</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Match Plan</string>
    </property>
   </widget>
   <widget class="QLabel" name="logoLabel">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>0</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_match">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>200</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start next match</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabelle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>641</width>
      <height>281</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Team</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Diff</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Goals</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Got</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Punkte</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="scorer">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>610</y>
      <width>421</width>
      <height>281</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Nummer</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Team</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Goals</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
